provide exit by escape key

Mallusca opened the pull request.

Mallusca committed:
added exit by escape key

AJIex1s commented on HeadsAndTails/Program.cs:
```diff
@@ -75,11 +75,14 @@ static void Main(string[] args)
 
                 Console.WriteLine();
-                Console.WriteLine("Нажмите Enter для начала новой игры...");
+                Console.WriteLine("Нажмите Enter для начала новой игры... Чтобы выйти нажмите Escape");
```
Split the message text into two lines

Mallusca replied:
[Fix](https://github.com/Mallusca/heads-and-tails/pull/4/commits/79da79dc683960572de06329bbc54ac309a890e5)

Mallusca committed:
fixed - split message into two lines

Mallusca committed:
fixed exit from the game with any button except Escape.

added 3 second delay by pressing Escape.
it means that you have delay after you pressing Escape.
and I think must be - 3 seconds.

AJIex1s commented on HeadsAndTails/Program.cs:
```diff
@@ -20,16 +20,27 @@ static void Main(string[] args)
+                    Console.Write(".");
+                    Thread.Sleep(300);
+                }
+                Environment.Exit(0);
```
You should use return here, instead of exit, return will terminate execution of main function. Replace and check it in all places.

Mallusca committed:
Replaced in all required places with return;
